### PR TITLE
Improve instructions for MacOS and demo

### PR DESCRIPTION
**Pull request by JulienBreux**
I just want to say thanks!

It's my tiny contrib'.

— JBX

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,22 @@ setting this up currently requires some effort:
 
 you should now be able to run the code in examples/
 
-# todo
+### For Mac OS
```
Comment (mjl-) [inline README.md:23]: do you have a reason to use brew and xquartz?
i'm not using xquartz, just plan9port that talks directly to macos. i would think that's a better experience than going through an x server on macos.

(also, i haven't done github PR's from this side yet, might take a while to get the hang of it)

Comment (JulienBreux) [reply README.md:23]: Hmm, my build of plan9port doesn't work without `--with-x11` with brew option :/

Comment (mjl-) [reply README.md:23]: have you tried building plan9port directly? that should be the most straightforward way to get it installed. and i can only imagine it's the only way recommended by plan9port. it should be as easy as cloning and running ./INSTALL.

**Comment by mjl-** [comment]
i've updated lots of docs, and this this PR no longer applies, so i'm closing it.
did you get plan9port itself working on macos?

**Pull request closed without merging**